nemo/collections/nlp/parts/nlp_overrides.py

```diff
@@ -229,3 +230,6 @@ def save_checkpoint(self, filepath, weights_only: bool):
                     )
                     atomic_save(checkpoint, filepath)
         return None
+
```

Comment:
Not required ? Is it a placeholder ?

Reply:
Were you referring to the comment below this?

Reply:
Yes

Reply:
Removed it for now. I want to do the NLPModel refactor in a seperate PR as this PR is getting too big already.